test/e2e/framework.go

```diff
@@ -190,6 +190,10 @@ type ExternalInfo struct {
 	vlanSubnetIPv6  string
 	vlanGatewayIPv6 string
 	vlanID          int
+
+	externalFRRIPv4 string
```

Comment:
Are these variables related to deploying an FRR container in Kind? I didn't find any usage of them in the kind script. I guess they are related to BGP e2e tests?

Reply:
> I didn't find any usage of them in the kind script. I guess they are related to BGP e2e tests?

Yes, they will be used in BGP e2e tests later.